domains/anees.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
```

```"owner"``` required to have ```username``` (your GitHub username) and ```email``` , please read here: https://docs.is-a.dev/domain-structure/#-owner-required

```suggestion
  "owner": {
      "username":"aneesnizam",
```